genesyscloud/tfexporter/genesyscloud_resource_exporter.go

```diff
@@ -1607,6 +1605,11 @@ func (g *GenesysCloudResourceExporter) resourceIdExists(refID string, existingRe
 }
 
 func (g *GenesysCloudResourceExporter) isDataSource(resType string, name string) bool {
+	for _, element := range resourceExporter.ExportAsData {
+		if element == resType+"::"+name || fetchByRegex(element, resType, name) {
```

Comment:
add this logic "element == resType+"::"+name || fetchByRegex(element, resType, name)" in a common method. since we are using the same for both the iterations of ExportAsData and  replaceWithDatasource